tools/meteor-services/deploy.js

```diff
@@ -321,6 +334,122 @@ function canonicalizeSite(site) {
   return parsed.hostname;
 };
 
+// Executes the poll to check for deployment success and outputs proper messages
+// to user about the status of their app during the polling process
+function executePollForDeploymentSuccess(versionId, deployPollTimeout, result) {
+  // Create a default polling configuration for polling for deploy / build
+  // In the future, we may change this to be user-configurable or smart
+  // The user can only currently configure the polling timeout via a flag
+  let pollingState = new PollingState(deployPollTimeout);
+  
+  var deploymentPollResult = buildmessage.enterJob({
```

Comment:
Can be const.  Note that const and let are basically the modern replacements for var. The main difference between let and var is that let has block scope whereas var has function scope.  (I recognize you're copying this line from somewhere else and that this codebase still has lots of var in it. Let's be modern for new stuff though!)

Reply:
👍